Include/pyerrors.h

```diff
@@ -122,6 +122,7 @@ PyAPI_DATA(PyObject *) PyExc_FileNotFoundError;
 PyAPI_DATA(PyObject *) PyExc_InterruptedError;
 PyAPI_DATA(PyObject *) PyExc_IsADirectoryError;
 PyAPI_DATA(PyObject *) PyExc_NotADirectoryError;
+PyAPI_DATA(PyObject *) PyExc_DirectoryNotEmptyError;
```

Comment:
Is there any reason not to sort by alphabetic order?
Looks like `PyAPI_DATA(PyObject *) PyExc_DirectoryNotEmptyError` should be below the  `PyAPI_DATA(PyObject *) PyExc_ConnectionResetError`